pallets/nfts/src/tests.rs

```diff
@@ -759,9 +775,10 @@ fn transfer_owner_should_work() {
 		assert_eq!(Balances::reserved_balance(&account(3)), 44);
 
 		assert_ok!(Nfts::transfer(RuntimeOrigin::signed(account(1)), 0, 42, account(2)));
-		// reserved_balance of accounts 1 & 2 should be unchanged:
+		// Reserve `balance_deposit` from account 2 and reserved balance of account 1 remains the
+		// same.
```

Comment:
Didn't look into why but the comment indicated that checks should be the case. You changed it to what was happening. The comment didn't provide any value but it is not the PR to improve unfortunately.
```suggestion
		// The reserved balance of account 1 should remain the same. For account 2 the `balance_deposit`  is 
                // reserved because it is the new item owner. 
```